src/index.js

```diff
@@ -1,3 +1,22 @@
+/** IE9, IE10 and IE11 requires all of the following polyfills. **/
```

Comment:
Babel-preset-env should take care of a lot of this I think. If the browser list stuff is set up. You might have to play with the `useBuiltIns` option though.